src/main/resources/model/instance.json

```diff
@@ -95,6 +95,20 @@
           "index": "keyword"
         }
       }
+    },
+    "publication": {
+      "type": "object",
+      "properties": {
+        "publisher": {
+          "inventorySearchTypes": "publisher",
```

Comment:
does it mean that user should use `publisher=="abcd"` CQL query?

Reply:
yes. that's right. I don't find search by publication in existing inventory search, but I guess it would be better to shorten name of field

Reply:
Yeah, shortening sound good for me, however it can be not so clear for users (most of them are expecting that properties matches json schema). We should document it somewhere (for example in module readme) if we're going to procced with the custom name.

 What do you think?

Reply:
I suppose that we can postpone a little bit a documenting of all features until current sprint search features will be completed and then create a single PR with whole options of search - for example: search by contributors looks like in CQL like ```(contributors =/@name "123") sortby title```